process/io/plot_proc.py

```diff
@@ -2807,8 +2716,7 @@ def parse_args(args):
     # Setup command line arguments
     parser = argparse.ArgumentParser(
         description="Produces a two page summary of the PROCESS MFILE output, using the MFILE.  "
-        "For info contact michael.kovari@ukaea.uk or james.morris2@ukaea.uk.  "
-        "If using PLASMOD you must specify the profile file using -m."
+        "For info contact michael.kovari@ukaea.uk or james.morris2@ukaea.uk."
```

Comment:
This would be a good opportunity to point people to issues/discussion/the list of contacts rather than using email (that may one day be outdated).

Reply:
Changed to the contact list on the front page